doctr/io/elements.py

```diff
@@ -215,7 +220,7 @@ class Page(Element):
     """Implements a page element as a collection of blocks
 
     Args:
-        blocks: list of block elements
+        blocks: Dictionary with list of block elements for each detection class
```

Comment:
I still try to find a solution to avoid this breaking change but i think in front of this a short explanation would help me to understand the use case for multi class in text detection or better what you want to reach with it :sweat_smile: 

Is it not possible to add another key value pair like` 'class_names': 'words'` instead of changing the output structure ?

Reply:
hmm something like a class_names is the same len as blocks and give the class name of each block ?

Reply:
Actually blocks are not resolved, so this sub object in the `Page` might be obsolete, we could either rename it to `Layer` to have a more meaningful page structure and keep it like that or add a `Layer` object between` Page` and `Block` in the hierarchy, so that a class would be on a layer and inside each layer we can resolve blocks and lines.